coala_quickstart/Extensions.py

```diff
@@ -0,0 +1,172 @@
+exts = {'.w': 'C',
+        '.cs': 'C#',
+        '.csx': 'C#',
+        '.c': 'C++',
+        '.c++': 'C++',
+        '.cpp': 'C++',
+        '.cxx': 'C++',
+        '.h': 'C++',
```

Comment:
We can do multiple languages per extension (see `.m` in the last commit). I'll modify :+1: